ui-v2/app/templates/dc/services/instance/proxy.hbs

```diff
@@ -14,32 +14,44 @@
           {{#if (env 'CONSUL_NSPACES_ENABLED')}}
             {{#if (not-eq item.DestinationType 'prepared_query')}}
               <li class="nspace">
-                {{or item.DestinationNamespace 'default'}}
-                <Tooltip>
-                  Namespace
-                </Tooltip>
+                <span>
+                  <Tooltip>
+                      Namespace
+                  </Tooltip>
+                </span>
+                <span>
+                  {{or item.DestinationNamespace 'default'}}
+                </span>
               </li>
             {{/if}}
           {{/if}}
             {{#if (and (not-eq item.Datacenter proxyDatacenter) (not-eq item.Datacenter ""))}}
               <li class="datacenter">
-                {{item.Datacenter}}
-                <Tooltip>
-                  Datacenter
-                </Tooltip>
+                <span>
+                  <Tooltip>
+                      Datacenter
+                  </Tooltip>
+                </span>
+                <span>
+                  {{item.Datacenter}}
+                </span>
```

Comment:
You know what I'm like with semantic-ness, but! I wonder if this pattern fits well with definition lists? You'd then be able to target `dt`'s in your CSS instead of using first-child etc. One of those no biggie ones